scripts/start/readCheckScenarioConfig.R

```diff
@@ -27,18 +27,27 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
             paste0(rownames(scenConf)[regionname], collapse = ", "),
             " – Titles with three capital letters or 'glob' may be confused with region names by magclass. Stopping now.")
   }
-  containsdot <- grep("\\.", rownames(scenConf))
-  if (any(containsdot)) {
-    warning("These titles contain a dot: ",
-            paste0(rownames(scenConf)[containsdot], collapse = ", "),
-            " – GAMS would not tolerate this, and quit working at a point where you least expect it. Stopping now.")
+  illegalchars <- grepl("[^[:alnum:]_-]", rownames(scenConf))
+  if (any(illegalchars)) {
+    warning("These titles contain illegal characters: ",
+            paste0(rownames(scenConf)[illegalchars], collapse = ", "),
+            " – Please use only [:alnum:]_- to avoid errors. Stopping now.")
```

Comment:
You and I know what `[:alnum:]` is, the median REMIND user does not …

Reply:
Better: `A-z0-9_-`?